runtime/rastrace/method_trigger.c

```diff
@@ -654,12 +697,12 @@ addTriggeredMethodSpec(J9VMThread *thr, const char *ptrMethodSpec, const struct
 		}
 
 		if (methodRule->entryAction != NULL && methodRule->entryAction->name != NULL
-		    && j9_cmdla_stricmp((char *)methodRule->entryAction->name, "jstacktrace") == 0) {
+			&& j9_cmdla_stricmp((char *)methodRule->entryAction->name, "jstacktrace") == 0) {
```

Comment:
If you want to improve this, the final `) {` goes on a line by itself, constants go on the left of `==` and `!=` and parentheses are missing:
```c
		if ((NULL != methodRule->entryAction)
			&& (NULL != methodRule->entryAction->name)
			&& (0 == j9_cmdla_stricmp((char *)methodRule->entryAction->name, "jstacktrace"))
		) {
```
Similar comment for lines 704-705.